.devcontainer/devcontainer.json

```diff
@@ -19,6 +19,13 @@
 		"ms-vscode.cmake-tools"
 	],
 
+	"features": {
+		"docker-from-docker": {
```

Comment:
What is this feature reference to?

Reply:
![image](https://github.com/usdot-jpo-ode/jpo-cvdp/assets/62157949/1f7a3e4d-bb82-4c94-ab24-f4e1246aa290)

There is a warning for this feature.
It also take 99331ms for the to run a port forwarding task. The devContainer terminal does not exit and return to user control.

It takes 359434 ms to run the vscode with Run: /usr/share/code/code.

After comment out the feature and rebuild. The terminal return quickly.
![image](https://github.com/usdot-jpo-ode/jpo-cvdp/assets/62157949/c7ee5d10-86f5-494b-ae91-f8e08600a741)